### ISSUE TITLE
Loading dump not working in multi-database setup 

### ISSUE THREAD

**Issue by foobear** [makandra/geordi#173]
In a Rails application with multiple databases, `geordi dump -l ... -d ...` fails with an error that I can not make any sense of.

```
$ geordi dump --load=tmp/example.dump --database=primary

# Sourcing dump into the  db
> Source file: tmp/example.dump
/home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/geordi-9.2.0/lib/geordi/dump_loader.rb:66:in `load': undefined method `_command' for #<Geordi::DumpLoader:0x0000560b63577c30 @dump_file="tmp/example.dump", @config={"default"=>{"adapter"=>"postgresql", "encoding"=>"unicode", "pool"=>5}, "development"=>{"primary"=>{"adapter"=>"postgresql", "encoding"=>"unicode", "pool"=>5, "database"=>"example_development"}, "data_migration"=>{"adapter"=>"postgresql", "encoding"=>"unicode", "pool"=>5, "database"=>"example_data_migration", "migrations_paths"=>[]}}, "test"=>{"primary"=>{"adapter"=>"postgresql", "encoding"=>"unicode", "pool"=>5, "database"=>"example_test"}, "data_migration"=>{"adapter"=>"postgresql", "encoding"=>"unicode", "pool"=>5, "database"=>"example_test_data_migration", "migrations_paths"=>[]}}}> (NoMethodError)
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/geordi-9.2.0/lib/geordi/commands/dump.rb:39:in `dump'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/geordi-9.2.0/exe/geordi:12:in `block in <top (required)>'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/geordi-9.2.0/lib/geordi/util.rb:14:in `installing_missing_gems'
	from /home/arne/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/geordi-9.2.0/exe/geordi:7:in `<top (required)>'
	from /home/arne/.rbenv/versions/3.0.3/bin/geordi:25:in `load'
	from /home/arne/.rbenv/versions/3.0.3/bin/geordi:25:in `<main>'
```

Is this a bug, or am I doing something wrong?

For reference, this is my `database.yml`:

```yaml
default: &default
  adapter: postgresql
  encoding: unicode
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>

development:
  primary:
    <<: *default
    database: example_development
  data_migration:
    <<: *default
    database: example_data_migration
    migrations_paths: [] # app migrations should not be run on this database

test:
  primary:
    <<: *default
    database: example_test<%= ENV['TEST_ENV_NUMBER'] %>
  data_migration:
    <<: *default
    database: example_test_data_migration<%= ENV['TEST_ENV_NUMBER'] %>
    migrations_paths: [] # app migrations should not be run on this database
```

**Comment by codener** [comment]
Multi-database support is currently only implemented for the server-side: you can select which database to dump, but not which one to load the dump into. We're implementing support in #174.

@foobear would you ever want to load a dump into a local database other than `primary`? 

/cc @kajatiger

**Comment by foobear** [comment]
> @foobear would you ever want to load a dump into a local database other than `primary`?

Probably not, but my primary database also is not called `primary` because of reasons.

For the sake of consistency, I suggest using the same command line switches / approach that exists for dumping the database.

**Comment by codener** [comment]
We're going with what you suggested: `geordi dump staging -d app -l` will dump the "app" database of staging, download it, and source it into the local "app" database.

**Comment by codener** [comment]
Released as 9.3.0.
